Track Your Reps/Views/RecentEventsVC.swift

```diff
@@ -0,0 +1,42 @@
+///
```

Comment:
I wanted to make a separate file / view for the Recent Events info, because the SummaryVC is getting bloated, but I failed to figure out how to add constraints for this subclassed UIView (or UIViewController)

So this file shouldn't be deleted, unless we can figure that out